inter-operator/architecture/basic.md

```diff
@@ -617,3 +618,8 @@ Now the setup is ready for taking requests. We depict this in the picture below.
 ## Runtime Flow
 After the interoperator is ready and setup across multiple clusters as described [above](#deployment-flow), service instance and service binding can be created. When in the master cluster, broker creates an `SFServiceInstance`, Scheduler picks it up first and schedules/assigns a cluster where service needs to be provisioned. Then [Service Instance Reconciler](#service-instance-reconciler) reconciles that `SFServiceInstance` in the sister cluster where it is scheduled. Once that is done, [provisioner](#provisioner) residing in the sister cluster takes over and from then onwards, the process described in [service provisioning](#service-fabrik-inter-operator-provisioner-1) is followed. For another `SFServiceInstance`, it is again scheduled in one of the sister cluster and provisioner provisions the service there. The picture below describes the steps.
 ![Inter-operator Runtime Flow](images/Runtime%20Flow%20Updated.png)
+
+## Limitations with Multi-Cluster deployment
+1. Interoperator currently does not take care of the cluster off-boarding.
+2. Service Operator in each sister cluster is assumed to be already deployed and version update-upgrade is managed/maintained by the service operator. Inter-operator does not do anything about it.
```

Comment:
```suggestion
2. Service Operator in each sister cluster is assumed to be already deployed and its version update/upgrade is managed/maintained by the service operator. Inter-operator does not do anything about it.
3. Interoperator does not take care of the Kubernetes and OS updates to the onboarded clusters.

Reply:
Also makes sense to add the below point, 
Service owners will have to monitor the clusters and their resource situations and add additional sister clusters if required

Reply:
Incorporated both the suggestions. Thanks @amshuman-kr @PadmaB !